lib/internal/cli_table.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const ansiRegex = () =>
+  new RegExp([
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*',
+    '(?:;[a-zA-Z\\d]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))',
+  ].join('|'), 'g');
+
+const { Buffer } = require('buffer');
+
+const TableChars = {
+  /* eslint-disable node-core/non-ascii-character */
+  middle_middle: '─',
+  row_middle: '┼',
+  top_right: '┐',
+  top_left: '┌',
+  left_middle: '├',
+  top_middle: '┬',
+  bottom_right: '┘',
+  bottom_left: '└',
+  bottom_middle: '┴',
+  right_middle: '┤',
```

Comment:
Can we please stick to our "normal" variable names and use camelCase instead of snake_case?